eng/pipelines/tagger.yml

```diff
@@ -0,0 +1,90 @@
+# Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+# Name: DotNet-Project-System-Tagger
+# URL: https://dev.azure.com/devdiv/DevDiv/_build?definitionId=17277
+
+# Creates a Git tag in our repo when a VS Insertion PR is merged.
+
+###################################################################################################################################################################
+# PIPELINE METADATA
+###################################################################################################################################################################
+
+# Disable CI builds for this pipeline.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/trigger?view=azure-pipelines#examples
+trigger: none
+
+# Disable PR builds for this pipeline.
+# See: https://docs.microsoft.com/azure/devops/pipelines/yaml-schema/pr?view=azure-pipelines#examples
+pr: none
+
+resources:
+  # https://stackoverflow.com/a/63270937/294804
+  repositories:
+  - repository: self
+    name: project-system
+    type: git
+    ref: feature/InsertionPRDescriptionAndTagging
```

Comment:
I need to remove this section as it was something I was trying to test with the trigger mechanism.

Reply:
This has been removed.